apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-description/edit.js

```diff
@@ -58,6 +56,9 @@ function SiteDescriptionEdit( {
 }
 
 export default compose( [
+	withDispatch( dispatch => ( {
```

Comment:
Is it worth looking at the new useDispatch hook (https://unfoldingneurons.com/2019/fantastic-hooks-and-where-to-use-them)? If we use this instead of withDispatch it can be imported within the useSiteOptions hook (I think that is the case ) so no need to wrap each fse block, and one less param to pass into useSiteOptions.